src/main/java/bgu/spl/net/impl/stomp/ConnectionsImpl.java

```diff
@@ -66,4 +66,10 @@ public void subscribe(String channel, int connectionId, int subscriptionId) {
     public void connect(ConnectionHandler<Frame> connectionHandler, int connectionId) {
         _connections.put(connectionId, connectionHandler);
     }
+
+    @Override
+    public void unsubscribe(String channel, int connectionId, int subscriptionId) {
+        // SimpleEntry is comparable and implements a deep compare
+        _channels.get(channel).remove(new AbstractMap.SimpleEntry<>(connectionId, subscriptionId));
```

Comment:
remove new? it's weird... you should explain it

Reply:
I am explaining it, SimpleEntry implements a deep compare so by deleting like this I am actually removing what I want to remove

Reply:
Maybe elaborate more